poc_mobile_verifier/src/cli/generate.rs

```diff
@@ -18,37 +21,62 @@ pub struct Cmd {
     #[clap(long)]
     before: NaiveDateTime,
     #[clap(long)]
-    input_bucket: String,
+    periods: i32,
```

Comment:
Maybe add some docstring for what the `periods` is supposed to represent

Reply:
removed so resolving